Python Tutorial Exercises(Week 1)/#3.py

```diff
@@ -0,0 +1,8 @@
+#3
+a = "harry had a little lamb"
+b = "ha" 
+
+for b in a:
+    z = a.count(b)
```

Comment:
As discussed in one of our previous sessions, for the purpose of this exercise, please change this  to not use `str.count()`